podpac/core/interpolation/selector.py

```diff
@@ -98,11 +98,11 @@ def select(self, source_coords, request_coords, index_type="numpy"):
         coords = []
         coords_inds = []
         for coord1d in source_coords._coords.values():
-            c, ci = self._select1d(coord1d, request_coords, index_type)
+            ci = self._select1d(coord1d, request_coords, index_type)
```

Comment:
Small refactor here, we weren't actually returning a modified coordinate so that was confusing before.